### DISCUSSION TITLE
2022 ASUS Zephyrus G14.

### DISCUSSION THREAD

**Discussion by prairiedad** [tshakalekholoane/bat#43] [General]
I'm afraid it doesn's seem to work on my G14. I downloaded pre-compiled v. 0.12-0-g8daf723, installed it in /usr/local/bin, set permissions to 744. It responds appropriately when given commands...version, threshold, persist...but does not seem to "survive." I just left it plugged in for a while, after having set threshold to 60, and returned to find the battery at 100%.  And yet it seemed to work fine for a few hours yesterday...

Any thoughts would be appreciated.

**Comment by prairiedad** [comment]
Just confirmed this. Set threshold to 60 via persist, but after booting into Windows and then back into Linux (in case I didn't make clear that I was dual-booting, that is the case) and threshold is back to 100.

Might it make any difference if I compile locally? I don't have golang installed, but can do so.

**Reply by tshakalekholoane** [reply]
I doubt compiling it locally will have an effect. Reapplying the threshold limit after booting is handled by systemd. What is the result of running the following command?

```shell
sudo systemctl status bat-boot.service
```

**Comment by prairiedad** [comment]
Absolutely my fault, apologies!

With compiling myself in mind, I had a look at the code, and realized that
it requires systemd, which by default my favorite distro (MX) doesn't use.
As soon as I rebooted with systemd enabled, the precompiled version worked
just fine, of course.

You might move the words "requires systemd" a little higher in the
description, but otherwise you've done a really nice job.

I'll probably run the systemd version of MX for a while to protect my new
G14's battery, but ultimately I'll take on an init script wrapper for the
necessary commands, so I can go back to SysVinit.

Many thanks for this utility, and again, my apologies.

Dave

**Reply by tshakalekholoane** [reply]
Cheers, I appreciate your feedback! It should probably support other init systems as well but perhaps that's something for another time.
